### PR TITLE
Exemplify the usage of MultiAdd

### PR DESCRIPTION
**Pull request by filipecosta90**
@K-Jo even tough we have examples on the tests, this example can also be present on the godoc.

### CODE REVIEW

**Comment by filipecosta90** [comment]
@gkorland can you also check this one? it is just documentation improvements

File: example_client_test.go [hunk 1]
```diff
@@ -102,3 +103,47 @@ func ExampleMultiGetWithOptions() {
...
+	labels1 := map[string]string{
+		"machine": "machine-1",
+		"az":      "us-east-1",
+	}
+	labels2 := map[string]string{
+		"machine": "machine-2",
+		"az":      "us-east-1",
+	}
+
+	err := client.CreateKeyWithOptions("timeserie-1", redistimeseries.CreateOptions{RetentionMSecs: 0, Labels: labels1})
```
Comment (gkorland) [inline example_client_test.go:125]: @filipecosta90 `RetentionMSecs: 0` without comment might be confusing, I would have remove it

File: example_client_test.go [hunk 2]
```diff
@@ -102,3 +103,47 @@ func ExampleMultiGetWithOptions() {
...
+	if err != nil {
+		log.Fatal(err)
+	}
+	err = client.CreateKeyWithOptions("timeserie-2", redistimeseries.CreateOptions{RetentionMSecs: 0, Labels: labels2})
```
Comment (gkorland) [inline example_client_test.go:129]: Same here

File: example_client_test.go [hunk 3]
```diff
@@ -102,3 +103,47 @@ func ExampleMultiGetWithOptions() {
...
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	// Adding multiple datapoints to multiple series
+	datapoints := []redistimeseries.Sample{{"timeserie-1", redistimeseries.DataPoint{1, 10.5}}, {"timeserie-1", redistimeseries.DataPoint{2, 40.5}}, {"timeserie-2", redistimeseries.DataPoint{1, 60.5}}}
```
Comment (gkorland) [inline example_client_test.go:135]: @filipecosta90 can you format it a bit better? perhaps each measurement should start a new line.

File: example_client_test.go [hunk 4]
```diff
@@ -102,3 +103,47 @@ func ExampleMultiGetWithOptions() {
...
+	timestamps, err := client.MultiAdd(datapoints...)
+
+	fmt.Println(fmt.Sprintf("Example adding multiple datapoints to multiple series. Added timestamps: %v", timestamps))
+
+	// Adding multiple datapoints to the same serie
+	datapointsSameSerie := []redistimeseries.Sample{{"timeserie-1", redistimeseries.DataPoint{3, 10.5}}, {"timeserie-1", redistimeseries.DataPoint{4, 40.5}}, {"timeserie-1", redistimeseries.DataPoint{5, 60.5}}}
```
Comment (gkorland) [inline example_client_test.go:141]: same here

Comment (filipecosta90) [reply example_client_test.go]: agreed and solved in the latest commit :+1:

File: example_client_test.go (hunk 3 above)
Comment (filipecosta90) [reply example_client_test.go]: agreed and solved in the latest commit :+1:

**Pull request merged**